integration/qbft/tests/utils.go

```diff
@@ -0,0 +1,178 @@
+package tests
+
+import (
+	"context"
+	"encoding/hex"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spec "github.com/attestantio/go-eth2-client/spec/phase0"
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	spectestingutils "github.com/bloxapp/ssv-spec/types/testingutils"
+	qbftstorage "github.com/bloxapp/ssv/ibft/storage"
+	"github.com/bloxapp/ssv/network"
+	"github.com/bloxapp/ssv/operator/validator"
+	protocolforks "github.com/bloxapp/ssv/protocol/forks"
+	protocolbeacon "github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+	protocolvalidator "github.com/bloxapp/ssv/protocol/v2/ssv/validator"
+	"github.com/bloxapp/ssv/protocol/v2/types"
+	"github.com/bloxapp/ssv/storage"
+	"github.com/bloxapp/ssv/storage/basedb"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"testing"
+	"time"
+)
+
+type Delay time.Duration
+type Slot phase0.Slot
+
+const (
+	NoDelay       = Delay(0)
+	OneRoundDelay = Delay(2 * time.Second)
+
+	DefaultSlot = Slot(phase0.Slot(spectestingutils.TestingDutySlot + 0)) //ZeroSlot
+)
+
+var (
+	KeySet4Committee  = spectestingutils.Testing4SharesSet()
+	KeySet7Committee  = spectestingutils.Testing7SharesSet()
+	KeySet10Committee = spectestingutils.Testing10SharesSet()
+	KeySet13Committee = spectestingutils.Testing13SharesSet()
+)
+
+type DutyProperties struct {
+	Slot  Slot
+	Idx   phase0.ValidatorIndex
```

Comment:
please name this field better. is it the duty index, validator index, an ID that is for property x?